### ISSUE TITLE
bolts.Task not found

### ISSUE THREAD

**Issue by riccardoch** [parse-community/Parse-SDK-Android#773]
I've updated Parse version (1.6.4), but when I try to compile I get this error:
```
Error:(59, 68) error: cannot access Task
class file for bolts.Task not found
```

**Comment by CoderickLamar** [comment]
Have you tried a clean build?

Additionally, the current build is: 1.16.5, please try that

**Comment by riccardoch** [comment]
With 1.6.5 version the compilation fails, for this reason I tried with 1.6.4, but the only one version that is correctly running on my project is 1.6.3 (except for push notifications on Android Oreo).

**Comment by pcegarra** [comment]
Same error, back to 1.6.3

**Comment by rogerhu** [comment]
I suspect the upgrade to Gradle 4 stopped publishing the bolts dependencies

https://github.com/parse-community/Parse-SDK-Android/blob/master/Parse/build.gradle#L59

**Comment by rogerhu** [comment]
Seems that way:

https://oss.jfrog.org/artifactory/oss-snapshot-local/com/parse/parse-android/1.16.5-SNAPSHOT/parse-android-1.16.5-20180113.212513-1.pom

**Comment by riccardoch** [comment]
Adding:

`api 'com.parse.bolts:bolts-tasks:1.4.0'`

in the gradle dependencies everything works.

Thanks @rogerhu

**Comment by rogerhu** [comment]
The maven-publish plugin doesn't publish dependencies.

Gradle4 appears to have broken the workaround I had:

**Comment by Jawnnypoo** [comment]
If we are just deploying to bintray now, we could switch to using this plugin: 

https://github.com/novoda/bintray-release

I would also throw in once again advocating for using jitpack as the official distribution source. It makes things super easy.

**Comment by rogerhu** [comment]
Does it generate the correct POM file?  Let's switch if it does.  

https://github.com/novoda/bintray-release/blob/7f554240896028f9abad01fef20f8b79809b4e03/core/src/test/groovy/com/novoda/gradle/release/TestGeneratePomTask.groovy#L51

**Comment by rogerhu** [comment]
Found the workaround: https://github.com/bintray/gradle-bintray-plugin/pull/202/files

**Comment by Jawnnypoo** [comment]
Ahh good, nice find

**Comment by rogerhu** [comment]
Issue fixed in 1.16.6 - https://bintray.com/parse/maven/download_file?file_path=com%2Fparse%2Fparse-android%2F1.16.6%2Fparse-android-1.16.6.pom
